scenes/game_elements/characters/enemies/guard/guard.tscn

```diff
@@ -5,7 +5,7 @@
 [ext_resource type="Texture2D" uid="uid://8t6ihmesvghq" path="res://scenes/game_elements/characters/enemies/guard/components/field_of_view.png" id="3_0hjcv"]
 [ext_resource type="Texture2D" uid="uid://c75ofhy3swhj3" path="res://scenes/game_elements/characters/enemies/guard/components/exclamation_mark.png" id="3_pnkgp"]
 [ext_resource type="Script" uid="uid://cxsi2xqcdyw7g" path="res://scenes/game_elements/characters/enemies/guard/components/light.gd" id="4_lptvm"]
-[ext_resource type="SpriteFrames" uid="uid://ovu5wqo15s5g" path="res://scenes/quests/story_quests/template/1_template_stealth/template_stealth_components/template_guard_enemy.tres" id="5_mswbt"]
+[ext_resource type="SpriteFrames" uid="uid://ovu5wqo15s5g" path="res://scenes/quests/story_quests/equipo2/1_equipo2_stealth/equipo2_stealth_components/equipo2_guard_enemy.tres" id="5_mswbt"]
```

Comment:
Here you have changed the default appearance of the guard in the template to be your guard. Please change this back to its previous value: `res://scenes/quests/story_quests/template/1_template_stealth/template_stealth_components/template_guard_enemy.tres`

and then, in your scene res://scenes/quests/story_quests/equipo2/1_equipo2_stealth/equipo2_stealth.tscn, change each guard's Sprite Frames property to your quest's version: `res://scenes/quests/story_quests/equipo2/1_equipo2_stealth/equipo2_stealth_components/equipo2_guard_enemy.tres`